setup.py

```diff
@@ -267,7 +267,7 @@ def initialize_options(self):
         self._clean_trees = []
         self._clean_exclude = ['np_datetime.c',
                                'np_datetime_strings.c',
-                               'period.c',
+                               'src/period.c',
```

Comment:
you could do that 
might be more clear
if you want to do that in that other pr that would be fine (and we can merge that first)